proto/zds.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+// GRPC package - part of the URL. Service is added.
+// URL: /PACKAGE.SERVICE/METHOD
+package istio.workload.zds;
+
+option go_package="pkg/zdsapi";
+
+enum Version {
+  NOT_USED = 0;
+  V1 = 1;
+}
+
+message ZdsHello {
```

Comment:
And if we start with version - seems another NIH protocol. It doesn't seem to need all this complexity - if we need to change we can use a different address.

Not a huge deal - but unnecessary IMO

Reply:
this was in response to https://github.com/istio/istio/pull/48253#issuecomment-1863792631